File: services/frontend-web/src/api/contracts/content/ContentResponse.ts
export interface ContentResponse {
  id: string;
  sourceType: string;
  rawText: string;
  createdAtUtc: string;
  originalUrl?: string | null;
  imageUrl?: string | null;
  summary: {
    title: string;
    shortSummary: string;
    keyPoints: string[];
    tags: string[];
    language: string;
    provider: string;
  };
}

File: services/backend-dotnet/TelegramAi.Backend.Api/Contracts/Content/ContentResponse.cs
namespace TelegramAi.Backend.Api.Contracts.Content;

public sealed record ContentResponse(
    Guid Id,
    string SourceType,
    string RawText,
    DateTimeOffset CreatedAtUtc,
    string? OriginalUrl,
    string? ImageUrl,
    ContentSummaryResponse Summary);
